scripts/doc/linked-xml-file.py

```diff
@@ -216,8 +216,13 @@ def dict_of_set_add(dos, key, value):
 <tt>
 """ % (pagename, fn))
 
-                xmlroot = ET.parse(filepath).getroot()
-                print_tags(xmlroot, 0, "prj", fh, None, 0, relfilepath)
+                try:
+                    xmlroot = ET.parse(filepath).getroot()
+                    print_tags(xmlroot, 0, "prj", fh, None, 0, relfilepath)
```

Comment:
Minor: Could be after the `except` block.
Optional: Is it possible to also print the line number? (Or is it already printed by `print(err)`?)
:fast_forward: